arelle/plugin/validate/ESEF/__init__.py

```diff
@@ -275,6 +318,7 @@ def checkFootnote(elt, text):
                        "http://www.xbrl.org/WGN/report-packages/WGN-2018-08-14/report-packages-WGN-2018-08-14.html: "
                        "%(documentSets)s (Document files appear to be in multiple document sets)"),
                 modelObject=doc, documentSets=", ".join(sorted(ixdsDocDirs)))
+        ixTargetUsage = val.authParam["ixTargetUsage"].upper() # logger requires upper case
```

Comment:
Instead of capitalizing the field here, why not just capitalize it in the json?

Reply:
done